test/tree-sitter-corpus.jl.faceup

```diff
@@ -0,0 +1,784 @@
+
+«c:9e10»
+«c:9E-1»
+«c:9f10»
+«c:9f-1»
+
+«c:.1e10»
+«c:1.1e10»
+«c:1.e10»
+
+«c:0x0123456789_abcdef.ABCDEFp0»
+«c:0x0123456789_abcdef_ABCDEF.p-1»
+«c:0x.0123456789_abcdef_ABCDEFp1»
+
+«x:# ==============================»
+«x:# character literals»
+«x:# ==============================»
+
+«c:' '»
+«c:'o'»
+«c:'\t'»
+«c:'\uffff'»
+«c:'\U10ffff'»
+
+«x:# ==============================»
+«x:# string literals»
+«x:# ==============================»
+
+«s:""»
+«s:"\""»
+«s:"foo
+ bar"»
+«s:"this is a \"string\"."»
+«s:"""this is also a "string"."""»
+«v:band» = «s:"Interpol"»
+«s:"»«:julia-ts-string-interpolation-face:$»«D:band»«s: is a cool band"»
+«s:"»«:julia-ts-string-interpolation-face:$(»«c:2»«D:π»«:julia-ts-string-interpolation-face:)»«s: is a cool number"»
+«s:"cells interlinked within »«:julia-ts-string-interpolation-face:$(»«D:"cells interlinked whithin »«:julia-ts-string-interpolation-face:$(»«D:"cells interlinked whithin one stem"»«:julia-ts-string-interpolation-face:)»«D:"»«:julia-ts-string-interpolation-face:)»«s:"»
+
+«x:# ==============================»
+«x:# command string literals»
+«x:# ==============================»
+
+«s:`pwd`»
+«s:m`pwd`»
+«s:`cd »«:julia-ts-string-interpolation-face:$»«D:dir»«s:`»
+«s:`echo \`cmd\``»
+«s:```
+echo "\033[31mred\033[m"
+```»
+
+«x:# ==============================»
+«x:# non-standard string literals»
+«x:# ==============================»
+
+«x:# FIXME: \s shouldn't be an escape_sequence here»
+«v:trailing_ws» = «s:r"\s+$"»
+«v:version» = «s:v"1.0"»
+«s:K"\\"»
+
+«x:# ==============================»
+«x:# comments»
+«x:# ==============================»
+
+«x:# comment»
+«x:#= comment =#»
+«x:#=
+comment
+=#»
+«v:x» = «x:#= comment =#» «c:1»
+
+«x:#=
+nested #= comments =# =#»
+«x:#==#»
+
+«x:# ==============================»
+«x:# assignment operators»
+«x:# ==============================»
+
+«v:a» = b
+a .. b = a * b
+«v:tup» = «c:1», «c:2», «c:3»
+«v:car», cdr... = list
+c &= d ÷= e
+
+«x:# ==============================»
+«x:# binary arithmetic operators»
+«x:# ==============================»
+
+a + b
+a ++ «c:1» × b ⥌ «c:2» → c
+a // b
+«v:x» = A \ (v × w)
+
+«x:# ==============================»
+«x:# other binary operators»
+«x:# ==============================»
+
+a & b | c
+(x >>> «c:16», x >>> «c:8», x) .& «c:0xff»
+
+Dict(b => c, d => e)
+
+x |>
+  f |>
+  g
+
+«c:1»..«c:10»
+(«c:1»:«c:10»...,)
+
+«x:# ==============================»
+«x:# binary comparison operators»
+«x:# ==============================»
+
+a === «c:1»
+a! != «c:0»
+
+A ⊆ B ⊆ C
+x ≥ «c:0» ≥ z
+
+«x:# ==============================»
+«x:# unary operators»
+«x:# ==============================»
+
+-A'
++a
+-b
+√«c:9»
+!p === !(p)
+«c:1» ++ +«c:2»
+
+«x:# =============================»
+«x:# operator broadcasting»
+«x:# =============================»
+
+a .* b .+ c
+.~[x]
+
+«x:# ==============================»
+«x:# ternary operator»
+«x:# ==============================»
+
+«v:x» = batch_size == «c:1» ?
```

Comment:
`julia-mode` uses `font-lock-keyword-face` for "?" and ":" in ternary expressions.

Reply:
Again, set `treesit-font-lock-level` to 4 to have it fontitified.